src/hooks/useWeather.jsx

```diff
@@ -11,9 +11,10 @@ export function useWeather(lat, lon) {
     }, [lat, lon]);
 
     async function fetchData(lat, lon) {
-        const apiKey = '642c1ad128fd816bcb4ea30d07064974';
+        const apiKey = process.env.REACT_APP_API_KEY;
+        const apiUrl = `${process.env.REACT_APP_API_URL}?lat=${lat}&lon=${lon}&appid=${apiKey}&units=metric`;
```

Comment:
create const folder (global) and export 

`export const {
  API_KEY,
  API_URL
} = Config`